samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/select/SelectAssessmentBean.java

```diff
@@ -52,6 +52,8 @@ public class SelectAssessmentBean implements Serializable {
 	private Boolean warnUserOfModification;
 	private boolean hasAnyAssessmentRetractForEdit = false;  // this is used to display the message on the bottom if there is any assessment retracted for edit.
 	private String displayAllAssessments = "2"; // display all
+	private String reviewAssessmentId;
```

Comment:
@Getter @Setter ?

Reply:
Didn't do that, since the whole file uses custom getters and setters